suite-native/transactions/src/components/transactionDetail/TransactionDetailAmount.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+import { Box, Card, Divider, Text } from '@suite-native/atoms';
+import { formatNetworkAmount } from '@suite-common/wallet-utils';
+import { WalletAccountTransaction } from '@suite-common/wallet-types';
+
+type TransactionDetailAmountProps = {
+    transaction: WalletAccountTransaction;
+};
+
+const TransactionDetailAmountRow = ({ title, value }: { title: string; value: string }) => (
+    <Box flexDirection="row" justifyContent="space-between">
+        <Text>{title}</Text>
+        <Text color="forest">{value}</Text>
+    </Box>
+);
+
+export const TransactionDetailAmount = ({ transaction }: TransactionDetailAmountProps) => {
+    const totalInput = formatNetworkAmount(
```

Comment:
Just a note: When I first saw it, I was like "yeah, I would prefer to have it directly on the component, it is a little bit crowded here before render :D":

`<TransactionDetailAmountRow title="Input" value={formatNetworkAmount(transaction.details.totalInput,
        transaction.symbol,
        true,
    ))} />`

but I don't have strong preferences for this...

Reply:
I would leave this as it is now because in future, when we will have more selectors implemented and they'll be flexible enough, I would prefer this to be selector itself where you just call this `formatNetworkAmount` function. I think this is fine for now. (It's pretty much same in suite-desktop).